setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="phcode42v2",
+    version="1.0.0",
+    description="Code42 for Phantom",
+    packages=find_packages(),
+    install_requires=["requests>=2.3",
+                      "pytest==4.4.0",
+                      "pytest-mock==1.10.3",
+                      "py42>=1.14.1",
```

Comment:
I think this will pull py42 (and phantomstubs, if something named that were to exist) from PyPI unless they already exist in the user's venv.

I think https://www.python.org/dev/peps/pep-0440/#direct-references can be used here to point to the wheel of py42 that we have in the repo, similar to this example: https://stackoverflow.com/a/65902365/12015509

Reply:
Why are we using the wheel instead of pypi?

Reply:
this is the only allowed way include dependencies in a shipped phantom app (we can use PyPI when running tests since the tests won't actually be included in the shipped app).

Reply:
Ahh nice I spent a bit of time trying to get it to use the wheel without success. This direct references functionality is just what I was looking for!